### PR TITLE
Update docs for recent changes.

### PR DESCRIPTION
**Pull request by vext01**
See commits.

### CODE REVIEW

File: src/dev/getting_started.md [hunk 1]
```diff
@@ -80,6 +75,11 @@ rustup override set yk-stage1
 Now `cargo` will run our compiler (also setting all of the various flags
 required) for this directory instead of the default Rust compiler.
 
+To select hardware tracing, we need to set an environment variable:
```
Comment (ltratt) [inline src/dev/getting_started.md:78]: s/we/you/ (to fit in with the rest of the file).

Comment (vext01) [reply src/dev/getting_started.md]: Fixed in 3163fea.

**Comment by vext01** [comment]
OK to squash?

**Comment by ltratt** [comment]
Please squash.

**Comment by vext01** [comment]
splat.

**Pull request merged**